packages/model-viewer/src/features/annotation.ts

```diff
@@ -192,30 +195,64 @@ export const AnnotationMixin = <T extends Constructor<ModelViewerElementBase>>(
 
       hotspot.updatePosition(config.position);
       hotspot.updateNormal(config.normal);
+      // force the annotation view to update
+      this[$annotationView].set('update', 1);
     }
 
     [$tick](time: number, delta: number) {
       super[$tick](time, delta);
-      this[$updateHotspots]();
-      this[$annotationRenderer].render(this[$scene], this[$scene].activeCamera);
+
+      // NOTE: checks the current annotation view against camera for
+      // position / field-of-view updates (or alternatively a forced update)
+      // QUESTION: is this already implemented elsewhere?
+      const position = this[$annotationView].get('position');
+      if (!position
+        || this[$annotationView].get('update')
+        || this[$scene].activeCamera.position.distanceTo(position) !== 0
+        || this[$scene].camera.getEffectiveFOV() !== this[$annotationView].get('fov')) {
```

Comment:
I think we need to run clang-format on these files. If you rebase, there is now a pre-commit hook that will do it for you.